terraform/main.tf

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+resource "juju_application" "nms" {
+  name  = "nms"
```

Comment:
Should we allow customizing app name via variables? It's mentioned in the README anyway.

Reply:
Done